### PR TITLE
remove max utils version restriction

### PR DESCRIPTION
**Pull request by risicle**
https://trello.com/c/wIs7vHwR

~I'm not convinced of the usefulness of this version restriction tbh...~ gone

### CODE REVIEW

**Comment by lfdebrux** [comment]
> I'm not convinced of the usefulness of this version restriction tbh...

Let's remove it then! ;)

**Comment by risicle** [comment]
I'm ok with that - can anyone think of any gotchas?

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
         'Markdown<3.0.0,>=2.6.7',
         'PyYAML<6.0,>=5.1.2',
         'inflection<1.0.0,>=0.3.1',
-        'digitalmarketplace-utils<49.0.0,>=44.0.1',
```
Comment (lfdebrux) [inline setup.py]: Does this library not require digitalmarketplace-utils at all?

Comment (katstevens) [reply setup.py]: It uses the date formatters (and the `DMSandboxedEnvironment` for handling `TemplateField` types).

Comment (lfdebrux) [reply setup.py]: I think this should be

```suggestion
        'digitalmarketplace-utils>=44.0.1',
```

**Pull request merged**